mobile/native/src/api.rs

```diff
@@ -110,7 +110,7 @@ pub async fn get_order(id: String) -> Result<Order> {
 
 #[tokio::main(flavor = "current_thread")]
 pub async fn get_orders() -> Result<Vec<Order>> {
-    let orders = order::handler::get_orders()
+    let orders = order::handler::get_orders_for_ui()
```

Comment:
nit: This naming convention is a bit strange. 😅 Could we place ui specifc requests in to a dedicated sub module so that it reads something like that?

```
let orders = order::handler::ui::get_orders();
```

or potentially only use some `.into()` magic?